From 217152dfb1be900455a886f03b896d7e68b20133 Mon Sep 17 00:00:00 2001
From: Jyri Sarha <jyri.sarha@nokia.com>
Date: Fri, 28 Aug 2009 13:12:09 +0300
Subject: [PATCH] bluetooth: Remove reference to audio_source_state

The fix from commit 8169a6a6c9 exppects A2DP source to be implemented
but we do not need it for fremantle.
---
 src/modules/bluetooth/bluetooth-util.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/modules/bluetooth/bluetooth-util.c b/src/modules/bluetooth/bluetooth-util.c
index 963ed28..3ed8ed5 100644
--- a/src/modules/bluetooth/bluetooth-util.c
+++ b/src/modules/bluetooth/bluetooth-util.c
@@ -657,7 +657,6 @@ static DBusHandlerResult filter_cb(DBusConnection *bus, DBusMessage *m, void *us
             /* Device will disconnect in 2 sec */
             d->audio_state = PA_BT_AUDIO_STATE_DISCONNECTED;
             d->audio_sink_state = PA_BT_AUDIO_STATE_DISCONNECTED;
-            d->audio_source_state = PA_BT_AUDIO_STATE_DISCONNECTED;
             d->headset_state = PA_BT_AUDIO_STATE_DISCONNECTED;
 
             run_callback(y, d, FALSE);
-- 
1.5.6.3

